013-quarkus-oidc-restclient/src/test/java/io/quarkus/qe/AbstractPingPongResourceTest.java

```diff
@@ -25,6 +29,14 @@
 
     AuthzClient authzClient;
 
+    @BeforeEach
+    public void setup() {
+        RestAssured.config = RestAssuredConfig.config()
+                .httpClient(HttpClientConfig.httpClientConfig()
+                        .setParam("http.socket.timeout", 1000)
+                        .setParam("http.connection.timeout", 1000));
```

Comment:
Constant should be on a static final constant.

Reply:
Updated